app/views/handlers/individual_response.py

```diff
@@ -73,24 +73,34 @@ def __init__(
         self._list_name = self._schema.get_individual_response_list()
 
         self._metadata = self._questionnaire_store.metadata
-        self.page_title = None
 
         if not self._is_location_valid():
             raise NotFound
 
-    def _is_location_valid(self):
-        self._list_name = self._schema.get_individual_response_list()
+    @cached_property
+    def _list_model(self):
+        return self._questionnaire_store.list_store[self._list_name]
 
-        list_model = self._questionnaire_store.list_store[self._list_name]
+    @cached_property
+    def _list_item_position(self):
+        return self._list_model.index(self._list_item_id)
```

Comment:
The person number is off by one. Using the list store `list_item_position` method would fix this.

Reply:
Done in 8e5d7c91